cmd/kind/build/nodeimage/nodeimage.go

```diff
@@ -75,10 +74,10 @@ func runE(flags *flagpole) error {
 		node.WithKuberoot(flags.KubeRoot),
 	)
 	if err != nil {
-		return errors.Wrap(err, "error creating build context")
+		return err
```

Comment:
if `errors.Wrap` is not used, then perhaps `fmt.Errorf(.....%v, err)` should be used, so that some of the extra error messaging is not lost?

Reply:
There's no extra info. We ran a build, it returns build errors.

Reply:
We wrap too much, wrapping should add context to errors that lack it (eg a file open error from the stdlib is not meaningful, what file?). If we wrap here we actually move the stack trace and context up away from the actual error.

I'll write a devel guide for this..

Reply:
compromise: instead we wrap all the things, but we only show stacktraces for V >0 (IE debugging mode)

dropping this commit